sdk/eventgrid/azure-eventgrid/samples/async_samples/sample_publish_cloud_event_using_dict_async.py

```diff
@@ -18,33 +18,55 @@
 import os
 import asyncio
 from azure.core.messaging import CloudEvent
-from azure.eventgrid.aio import EventGridClient
+from azure.eventgrid.aio import EventGridPublisherClient
 from azure.core.credentials import AzureKeyCredential
 
-topic_key = os.environ["EVENTGRID_CLOUD_EVENT_TOPIC_KEY"]
-endpoint = os.environ["EVENTGRID_CLOUD_EVENT_TOPIC_ENDPOINT"]
-
-
 async def publish():
+    # To Event Grid Basic
+    topic_key = os.environ["EVENTGRID_CLOUD_EVENT_TOPIC_KEY"]
+    endpoint = os.environ["EVENTGRID_CLOUD_EVENT_TOPIC_ENDPOINT"]
+
     credential = AzureKeyCredential(topic_key)
-    client = EventGridClient(endpoint, credential, level="Basic")
+    client = EventGridPublisherClient(endpoint, credential)
 
     # [START publish_cloud_event_dict_async]
     async with client:
-        await client.send(
-            [
-                {
-                    "type": "Contoso.Items.ItemReceived",
-                    "source": "/contoso/items",
-                    "data": {"itemSku": "Contoso Item SKU #1"},
-                    "subject": "Door1",
-                    "specversion": "1.0",
-                    "id": "randomclouduuid11",
-                }
-            ]
-        )
+        await client.send([
+            {
+                "type": "Contoso.Items.ItemReceived",
+                "source": "/contoso/items",	
+                "data": {	
+                    "itemSku": "Contoso Item SKU #1"	
+                },	
+                "subject": "Door1",	
+                "specversion": "1.0",	
+                "id": "randomclouduuid11"
+            }
+        ])
     # [END publish_cloud_event_dict_async]
 
+    # To Event Grid Namespaces
+    topic_endpoint = os.environ["EVENTGRID_ENDPOINT"]
+    topic_key = os.environ["EVENTGRID_KEY"]
+    topic_name = os.environ["EVENTGRID_TOPIC_NAME"]
+    sub = os.environ["EVENTGRID_EVENT_SUBSCRIPTION_NAME"]
+
+    credential = AzureKeyCredential(topic_key)
+    client = EventGridPublisherClient(topic_endpoint, credential, namespace_topic=topic_name)
+
+    async with client:
+        await client.send([
+        {
+            "type": "Contoso.Items.ItemReceived",
+            "source": "/contoso/items",	
+            "data": {	
+                "itemSku": "Contoso Item SKU #1"	
+            },	
+            "subject": "Door1",	
+            "specversion": "1.0",	
+            "id": "randomclouduuid11"
+        }
+    ])
 
-if __name__ == "__main__":
+if __name__ == '__main__':
```

Comment:
still need to run black maybe?

Reply:
hmm will double check generating should run black